contribute/overview.md

```diff
@@ -64,7 +64,7 @@ as possible for you to work in this repository. The following sections guide you
             <div class="component-icon">
                 <img src="/assets/images/spellcheck.svg" alt="Docker Desktop for Windows" width="45" height="45">
             </div>
-                <h2 id="docker-for-windows/install/"><a href="/contribute/style/terminology/">Docker terminology</a></h2>
+                <h2 id="docker-for-windows/install/"><a href="/contribute/style/terminology.md">Docker terminology</a></h2>
```

Comment:
These are HTML (`<a href= ...>`) links, which unlike the markdown links, don't get processed by Jekyll, so I don't think this change is correct